docs/src/api/decoratorcontroller.md

```diff
@@ -279,6 +281,18 @@ as opposed to:
 }
 ```
 
+
```

Comment:
This section and the one below are duplicated both here and in the CompositeController docs

I fear these will end up out of sync - but they share code, so should always be the same.

Could we split this out into a separate doc and reference from both?

Reply:
Good point, extracted